src/sentry/issues/endpoints/group_details.py

```diff
@@ -274,6 +274,8 @@ def get(self, request: Request, group: Group) -> Response:
                 }
             )
 
+            data["lifetime"]["count"] = get_group_global_count(group)
```

Comment:
after the front end is updated, ill update this endpoint so that the lifetime count is the only one that uses `get_group_global_count`